kvmagent/kvmagent/test/vm_plugin_testsuite/test_vm_sync.py

```
@@ -18,7 +24,7 @@ class TestVmPlugin(TestCase, vm_utils.VmPluginTestStub):
     def setUpClass(cls):
         network_utils.create_default_bridge_if_not_exist()
 
-    @misc.test_for(handlers=[
+    @env.test_for(handlers=[
```

Ruff (0.12.2)

`env` may be undefined, or defined from star imports

(F405)